dvc/render/match.py

```diff
@@ -36,14 +35,13 @@ def __init__(self, data: Dict):
     def group_definitions(self):
         groups = defaultdict(list)
         for rev, rev_content in self.data.items():
-            for config_file, config_file_content in (
+            for _config_file, config_file_content in (
                 rev_content.get("definitions", {}).get("data", {}).items()
```

Comment:
If we don't use `_config_file`, let's iter on `.values()`

```suggestion
            for config_file_content in (
                rev_content.get("definitions", {}).get("data", {}).values()
```